src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -192,9 +191,8 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
                     ticks: {
                         fontSize: 11,
                         callback: function(value:number) {
-                            return that.props.yAxis === "abs-count" ? value : (Math.round(value*10)/10) + '%';
+                            return _.round(value, 1) + (that.props.yAxis === "abs-count" ? '': '%');
```

Comment:
I changed this from what it was before because it was rounding, for example, 1.2 down to 1.  There are instances where the whole chart is only 2% tall and we need decimal granularity.

Reply:
Having `_.round(value, 1)` should make it so it rounds to 1 decimal place.  Without this function, labels sometimes appear as 0.40000000... or 0.6000000...